mix.exs

```diff
@@ -71,17 +72,28 @@ defmodule UAInspector.MixProject do
 
   defp docs do
     [
-      main: "UAInspector",
-      source_ref: "master",
-      source_url: @url_github
+      extras: [
+        "CHANGELOG.md",
+        "LICENSE": [title: "License"],
+        "README.md": [title: "Overview"]
+      ],
+      main: "readme",
+      source_url: @url_github,
+      source_ref: "v#{@version}",
+      skip_undefined_reference_warnings_on: ["CHANGELOG.md"],
+      formatters: ["html"]
     ]
   end
 
   defp package do
-    %{
+    [
       files: ["CHANGELOG.md", "LICENSE", "mix.exs", "README.md", "lib", "priv"],
-      licenses: ["Apache 2.0"],
-      links: %{"GitHub" => @url_github}
-    }
+      maintainers: ["Marc Neudert"],
```

Comment:
```suggestion
```

Hex is ignoring this field for quite some time now so there should be no benefit of adding it back.

Reply:
I assumed in https://github.com/hexpm/hex_core? TIL.